packages/core/src/part.mjs

```diff
@@ -78,6 +78,17 @@ Part.prototype.attr = function (name, value, overwrite = false) {
   return this
 }
 
+/**
+ * Gets a free ID to use in the part
+ *
```

Comment:
From the code that was previously removed, this line is missing in the comment: 
```
 * @param {string} prefix - An optional prefix to apply to the ID
```

Reply:
Added. Good catch